### DISCUSSION TITLE
인증토큰 관련 문의 입니다.

### DISCUSSION THREAD

**Discussion by hoksi** [commerce-api-naver/commerce-solutions#74] [묻고 답하기]
테스트 솔루션 등록후 SELLER 타입으로 oAuth2.0 인증 토큰 발급을 받기위해 노력중입니다.
권한 관련 오류가 발생하는데 무엇을 해야 할 지 모르겠습니다.

* trace id : cr3-150454-65772h^1732687548282^21867687
* reseponse 
```
{"code":"DataAccessDenied","message":"데이터에 대한 접근 권한이 없습니다.","timestamp":"2024-11-29T06:48:04.527555Z","invalidInputs":[{"name":"account_id","type":"NotAuthority.subscription","message":"subscription not found matching applicationId and accountId. CODE : SUBSCRIPTION_NOT_FOUND"}]}
```

**Comment by commerce-api-naver** [comment]
@hoksi 

> [!IMPORTANT]
> 게시해주신 내용은 커머스솔루션 개발사 연동 관련 문의로 판단되어 관리자가 게시 공간을 이동하였습니다.
> - 최초 게시된 공간: 커머스API 내 스토어 애플리케이션 기술지원 공간 (`/commerce-api`)
> - 이동된 공간: 커머스솔루션 개발사 기술지원 공간 (`/commerce-solutions`)

안녕하세요 커머스API 권현철입니다.

말씀하신 내용에 대해 답변 드리겠습니다.

'인증 토큰 발급 요청API' SELLER로 호출하기 위해서는 솔루션 구독자의 스마트스토어 계정 UID를 입력해야 정상호출이 가능합니다만, 말씀해주신 호출 건(`cr3-150454-65772h^1732687548282^21867687`)의 경우 판매자의 계정ID를 작성하여 해당 오류가 발생하였습니다.
솔루션을 구독중인 판매자의 스마트스토어 데이터에 접근을 하기 위함이라면 스마트스토어 계정 UID를 입력하여 '인증 토큰 발급 요청API' 호출 부탁드립니다. 혹, 솔루션 개발사 자기 자신(`"type:SELF"`)에 대한 인증 토큰 발급을 진행을 원하신다면 인증 토큰의 발급 유형(`type` 파라미터)를 변경하시어 API재호출 시도 부탁드립니다.

추가로 기본 연동 요소에 대한 자세한 내용은 하단의 커머스API센터 개발하기 문서를 참고 부탁드립니다.
- https://apicenter.commerce.naver.com/ko/basic/solution-doc#section/%EA%B0%9C%EB%B0%9C%ED%95%98%EA%B8%B0/%EA%B8%B0%EB%B3%B8-%EC%97%B0%EB%8F%99-%EC%9A%94%EC%86%8C-%EA%B0%80%EC%9D%B4%EB%93%9C 
- https://apicenter.commerce.naver.com/ko/basic/solution-doc#section/%EA%B0%9C%EB%B0%9C%ED%95%98%EA%B8%B0/%EC%86%94%EB%A3%A8%EC%85%98-%EA%B5%AC%EB%8F%85-%EC%97%B0%EB%8F%99-%ED%94%84%EB%A1%9C%EC%84%B8%EC%8A%A4

감사합니다.

**Comment by hoksi** [comment]
감사합니다.
확인하도록 하겠습니다.
